runbotics/runbotics-desktop/src/core/bpm/runtime/runtime.service.ts

```diff
@@ -376,12 +376,14 @@ export class RuntimeService implements OnApplicationBootstrap, OnModuleDestroy {
         };
 
         const services = this.createEngineExecutionServices(processInstanceId);
+        const triggerData = request.triggerData;
 
         const engineExecutionOptions: BpmnEngineExecuteOptions = {
             services,
             variables: {
                 ...request.variables,
                 tempFolder: this.getTempDirPath(),
+                ...(triggerData && 'userEmail' in triggerData && { userEmail: triggerData.userEmail }),
```

Comment:
if it works in case triggerData has no userEmail or even if triggerData does not exists at all than merge